<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureSearchEngine</class>
 <widget class="QDialog" name="TextureSearchEngine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Standard deviation treshold</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Square size</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0" rowspan="4">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mean treshold</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="squareSizeBox">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="meanTresSpinBox">
       <property name="value">
        <double>4.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="2">
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Search similar textures</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="stdTresSpinBox">
       <property name="value">
        <double>4.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QPushButton" name="resetButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="meanRGBSpinBox">
       <property name="value">
        <double>4.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mean RGB treshold</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
